test/test-server/kafka_rest_router.go

```diff
@@ -173,6 +173,7 @@ func handleKafkaRestTopics(t *testing.T) http.HandlerFunc {
 					"topic_name": "topic1",
 					"is_internal": false,
 					"replication_factor": 3,
+					"partitions_count": 6,
```

Comment:
Can you rewrite this so it isn't a big JSON string?

Reply:
How do i rewrite this? This string is being written into the http response: `io.WriteString(w, response)`